src/utils.ts

```diff
@@ -27,6 +27,22 @@ const deleteFolderRecursive = path => {
     }
 };
 
+// This is an annoying hack to get around an issue in Puppeteer
+// where the browser.close method hangs indefinitely
+// See https://github.com/Sparticuz/chromium/issues/85#issuecomment-1527692751
```

Comment:
Is this the link you meant to put here? It talks about fixing the `browser.close` issue, but not the way you're doing it here (apparently by disabling the Chrome GPU function instead).

Reply:
ah yes, it's at a different point in the thread. I'll update the link